wallet-new/src/Cardano/Wallet/WalletLayer.hs

```diff
@@ -569,6 +571,8 @@ data NewPaymentError =
     | NewPaymentTimeLimitReached TimeExecutionLimit
     | NewPaymentWalletIdDecodingFailed Text
     | NewPaymentUnknownAccountId Kernel.UnknownHdAccount
+    -- | NewPaymentAddressBadNetworkMagic expectedNetworkMagic invalidDestinationAddresses
```

Comment:
stray comment

Reply:
I actually put this comment on purpose in order to describe what the fields were intended for, but I suppose it could be removed.

Reply:
⬆️ while I find this sort of comment helpful, in that it gives you names for the constructor fields, it looks an awful lot like an additional constructor which was accidentally commented out. In this case, we can mostly infer what the fields are from the constructor name - there's a mismatch between Addresses and NetworkMagic, so we would reasonably guess that the fields are the NM and Addrs. I'd say we can drop the comment.

In other cases where it's less obvious what the comment means, there ought to be a way to make it more clear that it's a documentation line, not an accidentally commented-out line (but I don't have a good answer for what that is).